scripts_tf/create_label_map.py

```diff
@@ -0,0 +1,22 @@
+"""
+Create label map file.
+"""
+import os
+
+with open(
+    os.path.join(os.path.dirname(__file__), os.pardir, "config.yaml"), "r"
+) as stream:
```

Comment:
looks kinda weird imo, but it works

Reply:
Yeah, it's a bit awkward. Let's run with black for a little and if it's too cumbersome we can toss it.